os/fs/procfs/fs_procfslogsave.c

```diff
@@ -141,6 +142,16 @@ static ssize_t logsave_read(FAR struct file *filep, FAR char *buffer, size_t buf
 	return log_dump_read(buffer, buflen);
 }
 
+/****************************************************************************
+ * Name: logsave_write
+ ****************************************************************************/
+
+static ssize_t logsave_write(FAR struct file *filep, FAR const char *buffer, size_t buflen)
+{
+	log_dump_set_option(buffer, buflen);
```

Comment:
I think `return log_dump_set_option(buffer, buflen);` is right to return result value.